### PR TITLE
Alerting docs: refix sensu link

### PR DESCRIPTION
**Pull request by brendamuir**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by jdbaldry — APPROVED** [review]

File: .pnp.cjs
Comment (jdbaldry) [inline .pnp.cjs:1]: Actually, should this file be in the PR?

File: .pnp.loader.mjs
Comment (jdbaldry) [inline .pnp.loader.mjs:1]: Same for this one. Perhaps it should be removed from this PR?

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md
Comment (jdbaldry) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:1]: Was the addition of this page intended?

**Review by jdbaldry — CHANGES_REQUESTED** [review]

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md
Comment (brendamuir) [reply docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:1]: no -- why do my prs keep doing that? thats what happened to the backport one, too

File: .pnp.loader.mjs
Comment (brendamuir) [reply .pnp.loader.mjs:1]: how/why did these files get in there?

File: .pnp.cjs
Comment (brendamuir) [reply .pnp.cjs:1]: how do i delete them? i clicked delete file but they still seem to be there

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md
Comment (jdbaldry) [reply docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:1]: Are you running `git add .`? That adds all files in your worktree and its not always what you want. Instead, consider selectively adding only the files you intend to commit.

File: .pnp.cjs
Comment (jdbaldry) [reply .pnp.cjs:1]: You need to also click the "Commit changes" button after that to commit the deletion.

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md
Comment (brendamuir) [reply docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:1]: i use git add -p

Comment (jdbaldry) [reply docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:1]: Mm, in that case you should be able to choose not to add these extra patches. Maybe there is a git hook adding them if you aren't.

**Comment by brendamuir** [comment]
@jdbaldry think this one is ok then. didn't realise re the commit

**Pull request merged**